chain/events/filter/event.go

```diff
@@ -84,9 +83,6 @@ func (f *eventFilter) ClearSubChannel() {
 }
 
 func (f *eventFilter) CollectEvents(ctx context.Context, te *TipSetEvents, revert bool, resolver AddressResolver) error {
```

Comment:
Good stuff removing this completely from API calls that only need historical events.

Reply:
yeah, I realised there's a race in here - you could get an `Apply()` or `Revert()` between installing and removing it .. it'd be very very unlikely, and I don't think it'd be a big harm, but with this new functionality, if that were to happen, you could potentially get reverted events showing up in your query for historical events!